code/game/objects/effects/countdown.dm

```diff
@@ -157,3 +157,13 @@
 		var/obj/machinery/computer/arena/C = A.get_controller()
 		var/time_left = max(0, (C.start_time - world.time) / 10)
 		return round(time_left)
+
+/obj/effect/countdown/flower_bud
+	name = "flower bud countdown"
+
+/obj/effect/countdown/flower_bud/get_value()
+	var/obj/structure/alien/resin/flower_bud_enemy/bud = attached_to
+	if(!istype(bud))
+		return
+	else if(bud.growth_timer)
+		return round(timeleft(bud.timerid), 1)
```

Comment:
```
code\game\objects\effects\countdown.dm:169:error: bud.timerid: undefined var
```
Did you mean to use bud.growth_timer instead?

Reply:
yeah, but it definitely doesn't damn work even if the vars are right. hummm...